tools/topology/sof/pipe-demux-playback.m4

```diff
@@ -46,7 +46,7 @@ W_MUXDEMUX(0, 1, PIPELINE_FORMAT, 2, 2, LIST(`         ', "DEMUX"))
 
 # Low Latency Buffers
 W_BUFFER(0, COMP_BUFFER_SIZE(2,
-	COMP_SAMPLE_SIZE(PIPELINE_FORMAT), PIPELINE_CHANNELS, eval(PCM_MAX_RATE / SCHEDULE_PERIOD)),
+	COMP_SAMPLE_SIZE(PIPELINE_FORMAT), PIPELINE_CHANNELS, COMP_BUFFER_FRAMES(PCM_MAX_RATE, SCHEDULE_PERIOD)),
```

Comment:
@juimonen just wondering on line 52 of this file for the W_BUFFER(1...), there's a val(). is this a typo?

Reply:
yes, thanks for pointing out. updated now.